### PR TITLE
add examples/state-scripts

### PR DESCRIPTION
**Pull request by mirzak**
Background, https://tracker.mender.io/browse/MEN-2457

### CODE REVIEW

File: examples/state-scripts/wait-for-network/Sync_Enter_00 [hunk 1]
```diff
@@ -0,0 +1,934 @@
+
+
+
+
+
+
+<!DOCTYPE html>
+<html lang="en">
+  <head>
```
Comment (kacf) [inline examples/state-scripts/wait-for-network/Sync_Enter_00]: Did you forget to click 'raw' on this one? :-)

Comment (mirzak) [reply examples/state-scripts/wait-for-network/Sync_Enter_00]: Doh :D

Comment (mirzak) [reply examples/state-scripts/wait-for-network/Sync_Enter_00]: Fixed

**Comment by kacf** [comment]
Merged in #403.

**Pull request closed without merging**